### ISSUE TITLE
support composer

### ISSUE THREAD

**Issue by JasonBoyles** [rackspace-orchestration-templates/php-app-single#17]
From Jamie Hannaford:
> The vast majority of PHP apps these days use Composer for dependency management.
> It’d be great if we could have an added bool parameter in the template file that executed 
> something like this once the PHP app dir has been set up:
>
> `curl -Ss https://getcomposer.org/installer | php`
> `mv composer.phar /usr/bin/composer`
> `composer install`

**Comment by JasonBoyles** [comment]
[Composer](https://getcomposer.org) deps are specified in a [`composer.json` file](https://getcomposer.org/doc/00-intro.md#declaring-dependencies) in the root of the app source tree. We can key off this and only run a `composer install` if that file is present.

**Comment by jamiehannaford** [comment]
sounds good :+1:

**Comment by jamiehannaford** [comment]
@JasonBoyles same question: has there been any progress on this? :)

**Comment by JasonBoyles** [comment]
@jamiehannaford No. We have assigned tasks that are higher priority right now. I promise you I will update this issue once we get more of our backlog behind us.

**Comment by jamiehannaford** [comment]
@JasonBoyles Any update on this?

**Comment by rs-randallburt** [comment]
@jamiehannaford @chrishultin is taking over template maintenance. Pinging him to take a look.
